content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -208,54 +92,12 @@ To install Minikube manually on Windows, download [`minikube-windows-amd64`](htt
 
 {{% /capture %}}
 
-## Confirm Installation
-
-To confirm successful installation of both a hypervisor and Minikube, you can run the following command to start up a local Kubernetes cluster:
-
-{{< note >}}
+## Starting your first cluster
 
-For setting the `--driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--driver` values is available in [specifying the VM driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
-
-{{< /note >}}
+From a terminal with administrator access (but not logged in as root), run:
 
-```shell
-minikube start --driver=<driver_name>
-```
-
-Once `minikube start` finishes, run the command below to check the status of the cluster:
-
-```shell
-minikube status
-```
-
-If your cluster is running, the output from `minikube status` should be similar to:
-
-```
-host: Running
-kubelet: Running
-apiserver: Running
-kubeconfig: Configured
-```
-
-After you have confirmed whether Minikube is working with your chosen hypervisor, you can continue to use Minikube or you can stop your cluster. To stop your cluster, run:
-
-```shell
-minikube stop
-```
-
-## Clean up local state {#cleanup-local-state}
-
-If you have previously installed Minikube, and run:
 ```shell
 minikube start
 ```
 
-and `minikube start` returned an error:
-```
-machine does not exist
-```
-
-then you need to clear minikube's local state:
-```shell
-minikube delete
-```
+If minikube fails to start, see the [minikube drivers page](https://minikube.sigs.k8s.io/docs/docs/drivers) for help setting up a compatible container or virtual-machine manager.
```

Comment:
Should we direct users to create issues in the k/minikube repo? I have seen a few issues created in the k/website repo which then don't get prompt responses. Actually there is a "Create Issue" button at the bottom of the page that leads to k/website repo.

Reply:
We could also point users to Discourse / Slack etc as places to ask for advice. Maybe.